Yank/pipeline.py

```diff
@@ -1253,10 +1253,13 @@ def _setup_molecules(self, *args):
                 # Generate parameters
                 charge_method = mol_descr['antechamber']['charge_method']
                 input_mol_path = os.path.abspath(mol_descr['filepath'])
+                # Use Gaff in parameters, otherwise default to gaff2
+                gaff = 'gaff' if 'leaprc.gaff' in mol_descr['leap']['parameters'] else 'gaff2'
```

Comment:
We had a discussion with David Mobley about this, and he feels strongly about preferring gaff against gaff2, so I'm not sure which one we should use as default.